src/Microsoft.TemplateEngine.Utils/MultiValue.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.TemplateEngine.Utils
+{
+    public class MultiValue
```

Comment:
documentation might be useful, as the member is public

Reply:
just thinking aloud on better name for this class: what about `ParameterMultiValue`?